content-src/components/DiscoveryStreamComponents/CardGrid/_CardGrid.scss

```diff
@@ -1,11 +1,23 @@
+$col4-header-line-height: 20;
+$col4-header-font-size: 13;
```

Comment:
I don't see this actually being set to the font size, just added to the limit-visibile-lines mixin.

Reply:
That mixin now takes the font size for some internal calculations and emits it as well.